website/docs/sql-function-glossary/sql-comments.md

```diff
@@ -0,0 +1,58 @@
+---
+id: sql-comments
+title: SQL Comments
+description: A CTE is a temporary result set that can be used in a SQL query. You can think of a CTE as a separate, smaller query within the larger query you’re building up.
+---
+
+<head>
+    <title>Working with SQL comments</title>
+</head>
+
+SQL comments…a two-folded thing: Are we talking about comments inline in SQL? Or comments on a table or view in the database?
+
+Why not both!
+
+In this page, we’ll unpack how to create both inline and database object-level comments, general best practices around SQL comments, and how dbt can help you improve (and version-control) your comments.
+
+## How to create SQL comments
+
+Inline SQL comments will begin with two dashes (--) in front of them in a query or dbt model; any text following these dashes is therefore what you’d call “commented out.” For longer, multi-line comments, you’ll typically see this syntax `/* your multi-line comment here */` used.
```

Comment:
Should we explain what happens when something is "commented out" (aka it won't run when you execute the query)

Reply:
yeah I see that

Reply:
Is this something I can help with?

Reply:
We can add this line after the final sentences: "When line(s) are commented out in SQL, they are ignored by the query engine."